src/components/Panel/Panel.tsx

```
@@ -0,0 +1,55 @@
+  };
+
+  return (
+    <UIKitPanel
+      {...rest}
+      ref={ref}
+      resize={resize}
+      contentContainerRef={handleContainerRef}
+    >
+      {children}
+      {resize === 'vertical' && (
+        <div className={s.resizeHandle} onMouseDown={onMouseDown} />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)